hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AReadOpContext.java

```diff
@@ -18,10 +18,11 @@
 
 package org.apache.hadoop.fs.s3a;
 
+import java.util.concurrent.ExecutorService;
```

Comment:
There's a `javax.annotation.Nullable` import below Apache. Can you bring it up to this import?

rel: https://github.com/steveloughran/formality/blob/master/styleguide/styleguide.md#imports